source/transfers/contracts/TransferHandlerRegistry.sol

```diff
@@ -0,0 +1,58 @@
+pragma solidity 0.5.12;
+
+import "./interfaces/ITransferHandler.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
```

Comment:
Did we want to make this class Ownable and the subsequent methods onlyOwner?

Reply:
ownable it removed per dmosites Removing Ownable and syntax update"

Reply:
removed